test/state/precompiles.cpp

```diff
@@ -0,0 +1,25 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2022 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "precompiles.hpp"
+
+namespace evmone::state
+{
+using namespace evmc::literals;
+
+std::optional<evmc::Result> call_precompile(evmc_revision rev, const evmc_message& msg) noexcept
+{
+    if (evmc::is_zero(msg.code_address) || msg.code_address > 0x09_address)
```

Comment:
Why have `is_zero`? Wouldn't `> 0x09_address` be enough?

Reply:
There is no precompile at address 0.

Reply:
Oh right.